lib/index.ts

```diff
@@ -172,6 +185,14 @@ export class Compiler {
       "};\n" +
       "export default exportedTypeSuite;\n";
   }
+  private _compileIndexSignatureDeclaration(node: ts.IndexSignatureDeclaration): string {
+    if (this.options.ignoreIndexSignature) {
+      return ignoreNode;
```

Comment:
I've tried with empty string but in that case is generating a statement line with a comma in the middle.

Reply:
Oh I see. In that case I suggest returning an actual empty string and just handling that specially (i.e. skipping the comma) -- I think that's a bit cleaner than returning text like "IGNORE", so that no one has to wonder whether such text might ever be a valid value.

Reply:
I initially used an empty string but was like using "null" value, so I preferred a constant value with a meaningful name.

Reply:
Strings are the intended outputs of these functions, so if we want a value that represents "no value", I think an empty string is a good choice. And since we are already returning the empty string from compileNode() in one case, it's also a more consistent choice.

Reply:
OK. If I change the constant value to "" do you think would be OK? We will have a meaningful name and the empty string.

Reply:
I wrongly commit the proposed solution before your answer. Let me know if it is not OK and I will rollback.